src/templates/edit.tsx

```diff
@@ -58,6 +60,20 @@ const Edit: () => JSX.Element = () => {
   const [saveState, setSaveState] = useState<SaveState>();
   const [saveStateFetched, setSaveStateFetched] = useState<boolean>(false); // needed because saveState can be empty
   const [devPageSets, setDevPageSets] = useState<any>(undefined);
+  const [parentStatus, setParentStatus] = useState<ParentStatus>("pending");
```

Comment:
These changes are going to have to move to the visual-editor lib and not be in the starter once Brian merges his stuff.